fili-core/src/main/java/com/yahoo/bard/webservice/async/jobs/stores/ApiJobStoreFilterOperation.java

```diff
@@ -0,0 +1,10 @@
+// Copyright 2016 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE file distributed with this work for terms.
+package com.yahoo.bard.webservice.async.jobs.stores;
+
+/**
+ * Types of legal ApiJobStoreFilter operations.
+ */
+public enum ApiJobStoreFilterOperation {
+    eq
```

Comment:
Why are we using our own special filter operations instead of the `FilterOperation` enum? We might as well support the full filter syntax, even if the full filter semantics isn't necessary supported by every `ApiJobStore`.